.phan/config.php

```diff
@@ -31,7 +31,10 @@
     "suppress_issue_types" => [
         "PhanUnusedPublicNoOverrideMethodParameter",
         // Until phan/phan#4746 is fixed
-        "PhanTypeMismatchArgumentInternal"
+        "PhanTypeMismatchArgumentInternal",
+        "PhanUndeclaredMethod",
```

Comment:
Why are these being added? If we've already fixed this class of errors they shouldn't be re-added.